core/src/avm2/globals/flash/display/Screen.as

```diff
@@ -0,0 +1,66 @@
+package flash.display
+{
+  import __ruffle__.stub_getter;
+  import flash.geom.Rectangle;
+
+  [API("661")]
+  public final class Screen
+  {
+    private static var _screens:Array;
+    private var _width:int;
+    private var _height:int;
+
+    [API(661)]
```

Comment:
I don't think this needs a versioning, since the entire Screen class is already versioned